### PR TITLE
Comparison with PML in FAQ

### PR DESCRIPTION
**Pull request by AlekseySh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +48,49 @@ Well, it makes sense as a starting point. But there are several possible drawbac
...
+  which allows training models by preparing a config and your data in the required format
+  (it's like converting data into COCO format to train a detector from [mmdetection](https://github.com/open-mmlab/mmdetection)).
+
+* OML has config based examples on popular benchmarks close to real life (like photos of products of thousands ids).
+  We found some good combinations of hyperparameters on these datasets, trained and published models and their configs.
+  Thus, it makes OML more recipes oriented than PML, and it's author
+  [confirms](https://github.com/KevinMusgrave/pytorch-metric-learning/issues/169#issuecomment-670814393)
+  this saying that his library is a set of tools rather the recipes, moreover, the examples in PML are mostly for CIFAR and MNIST datasets.
+
+* OML has the [Zoo](https://github.com/KevinMusgrave/pytorch-metric-learning) of pretrained models that can be easily accessed from
```
Comment (DaloroAT) [inline README.md:69]: bad url))

Comment (AlekseySh) [reply README.md:69]: thx

File: README.md [hunk 2]
```diff
@@ -48,6 +48,49 @@ Well, it makes sense as a starting point. But there are several possible drawbac
...
+before started working on our project we tried to use it, but in the end, we decided to work on our pipeline.
+Here is our opinion about the advantages of OML:
+
+* OML has [Config API](https://open-metric-learning.readthedocs.io/en/latest/examples/config.html)
+  which allows training models by preparing a config and your data in the required format
+  (it's like converting data into COCO format to train a detector from [mmdetection](https://github.com/open-mmlab/mmdetection)).
+
+* OML has config based examples on popular benchmarks close to real life (like photos of products of thousands ids).
+  We found some good combinations of hyperparameters on these datasets, trained and published models and their configs.
+  Thus, it makes OML more recipes oriented than PML, and it's author
```
Comment (b0nce) [inline README.md:65]: "its author" instead of "it's author"

File: README.md [hunk 3]
```diff
@@ -48,6 +48,49 @@ Well, it makes sense as a starting point. But there are several possible drawbac
...
+  [confirms](https://github.com/KevinMusgrave/pytorch-metric-learning/issues/169#issuecomment-670814393)
+  this saying that his library is a set of tools rather the recipes, moreover, the examples in PML are mostly for CIFAR and MNIST datasets.
+
+* OML has the [Zoo](https://github.com/KevinMusgrave/pytorch-metric-learning) of pretrained models that can be easily accessed from
+  the code in the same way as in `torchvision` (when you type `resnet50(pretrained=True)`).
+
+* OML is integrated with [PyTorch Lightning](https://www.pytorchlightning.ai/), so, we can use the power of its
+  [Trainer](https://pytorch-lightning.readthedocs.io/en/stable/common/trainer.html).
+  This is especially helpful when we work with DDP, so, you simply compare our
+  [DDP example](https://open-metric-learning.readthedocs.io/en/latest/examples/python.html)
```
[the remark this answers is not in the document]
Comment (AlekseySh) [reply README.md:77]: i think it will be too much if we list all of the features of PL trainer

File: README.md [hunk 4]
```diff
@@ -48,6 +48,44 @@ Well, it makes sense as a starting point. But there are several possible drawbac
...
 
 
+<details>
+<summary>What is the difference between Open Metric Learning and PyTorch Metric Learning?</summary>
+<p>
+
+[PML](https://github.com/KevinMusgrave/pytorch-metric-learning) is the popular library for Metric Learning
+and it includes rich collection of losses, miners, distances, and reducers; that is why we provide straightforward
+[examples](https://github.com/OML-Team/open-metric-learning#usage-with-pytorch-metric-learning) of using them with OML.
+Initially, we tried to use PML, but in the end, we came up with our library, which is more pipeline / recipes oriented.
```
Comment (b0nce) [inline README.md:58]: Probably we need some sentence before list of advantages, like: "Here's what OML is capable of:" or "That is how OML differs from PML:" or something else.

**Review by DaloroAT — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (AlekseySh) [reply README.md:58]: thx

**Pull request merged**